tests/ci/labels/features.go

```diff
@@ -0,0 +1,20 @@
+package labels
+
+import (
+	. "github.com/onsi/ginkgo/v2"
```

Comment:
I think we should skip using dot import but give it an alias, e.g. `g "github.com/onsi/ginkgo/v2"`

Reply:
What is the motivation for not using dot import? 
Just curious as we tend to use dot alias a lot for ginkgo?

Reply:
I would keep the dot aliases and it pollutes less the code (more readable) and we do know where those methods come from.

Reply:
Just anxious dot import may cause the warning message `should not use dot imports (ST1001)` by go-staticcheck. If it could be passed the glangcli-lint check, it could be fine to me.

Reply:
@yasun1 I understand your concern. I think we could fix/change that later.
I created https://issues.redhat.com/browse/OCM-6708 to fix lint issues and that could be done as part of it.